.buildkite/pipeline_perf.py

```diff
@@ -73,7 +73,7 @@ def build_group(test):
         devtool_opts += " --ab"
         pytest_opts = f" {ab_opts} run {REVISION_A} {REVISION_B} --test {test_path}"
     else:
-        pytest_opts += f" -m nonci {test_path}"
+        pytest_opts += f" -m '' {test_path}"
```

Comment:
Doesn't this mean that nonci tests will not be collected?

Reply:
Passing an empty string to `-m` (somewhat unintuitively) means "collect all tests, ignoring markers". We already use this trick in `tools/ab_test.py` to get it to run both nonci and non-nonci tests.

Reply:
True, added a comment